test/timeout.ts

```diff
@@ -273,9 +273,9 @@ test.serial('connect timeout (ip address)', withServerAndFakeTimers, async (t, _
 	);
 });
 
-test.serial('secureConnect timeout', withServerAndFakeTimers, async (t, _server, got, clock) => {
+test.serial('secureConnect timeout', withHttpsServer({}, true), async (t, _server, got, clock) => {
```

Comment:
Avoid this. It has ambiguous meaning. Does the `true` mean that we're using the HTTPS server? What if it's false? Does it mean that the HTTPS is turned off?

It would be best to include it as an option inside the object (first argument).

Reply:
https://github.com/sindresorhus/got/pull/1461#issuecomment-691153647